test/src/androidTest/kotlin/io/realm/shared/notifications/RealmListNotificationsTests.kt

```diff
@@ -33,38 +47,207 @@ class RealmListNotificationsTests : NotificationTests {
     }
 
     @Test
-    @Ignore
     override fun initialElement() {
-        TODO("Waiting for RealmList support")
+        val dataSet = OBJECT_VALUES
+
+        realm.writeBlocking {
+            val managedContainer = copyToRealm(RealmListContainer())
+            managedContainer.objectListField
+                .addAll(dataSet)
```

Comment:
Nitpick: Why not on the above line as above?

Reply:
I personally find it easier to have an overview of the entire call when things are in separate lines, my eye-brain interaction is much smoother 🤷 I think this may look/feel weird in case of accessing properties instead of method calls though.